lab-jonathan/lib/response.js

```diff
@@ -0,0 +1,21 @@
+let response = {};
```

Comment:
No need for this file any longer, since express is taking care of things for you.